src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -103,46 +103,62 @@ public async Task<IActionResult> Applications()
             var applications = await _apiClient.GetApplications(signinId, false);
             applications = applications.Where(app => app.ApplicationStatus != ApplicationStatus.Rejected).ToList();
 
-            if (!applications.Any())
+            var application = new Apply();
+            Guid applicationId;
+            string applicationStatus;
+
+            if (applications.Any())
```

Comment:
I know it's existing code, but we shouldn't be using Any() here as it's a List and therefore should use Count

Reply:
Sorted